ts/model_loader.py

```diff
@@ -74,48 +76,71 @@ def load(self, model_name, model_dir, handler, gpu_id, batch_size):
             with open(manifest_file) as f:
                 manifest = json.load(f)
 
+        function_name = None
         try:
-            temp = handler.split(":", 1)
-            module_name = temp[0]
-            function_name = None if len(temp) == 1 else temp[1]
-            if module_name.endswith(".py"):
-                module_name = module_name[:-3]
-            module_name = module_name.split("/")[-1]
-            module = importlib.import_module(module_name)
-            # pylint: disable=unused-variable
-        except ImportError as e:
-            module_name = ".{0}".format(handler)
-            module = importlib.import_module(module_name, 'ts.torch_handler')
-            function_name = None
+            module, function_name = self._load_handler_file(handler)
```

Comment:
@abishekchiffon Why has the logic for handling multiple files been removed?

Reply:
The logic has not been removed. It has been moved to the function _load_handler_file of the same TsModelLoader class.